frontend/src/employee-frontend/components/unit/unit-details/UnitEditor.tsx

```diff
@@ -205,12 +208,16 @@ const emptyTimeRange: JsonOf<TimeRange> = {
 
 const emptyOperationWeek = [null, null, null, null, null, null, null]
 
-type EditableTimeRange = JsonOf<TimeRange> | null
+type EditableTimeRange = JsonOf<TimeRange>
 
 type FormErrorItem = { key: string; text: string }
 
 type UnitEditorErrors = {
-  rangeErrors: RangeValidationResult[]
+  rangeErrors: {
```

Comment:
this whole file is starting to be a bit of a mess, could be refactored to use form framework at some point, but maybe not right now